### ISSUE TITLE
Facilitate quick building of trust

### ISSUE THREAD

**Issue by rreusser** [msftenhanceprovenance/msftenhanceprovenance#1]
Hi, thanks for doing what you're doing! I get sick of making these PRs, so I very much appreciate that it caught a missed link in a repo I maintain.

To be perfectly honest though, the opening line, "This is not a bot," had the unintended consequence of making me *very* suspicious this was a malicious bot. I examined the PR I received very carefully for typos, Cyrillic characters, or other subtle tricks. I'm [not entirely alone](https://mobile.twitter.com/kuvos/status/1411969700947542018?lang=bg).

The thing which made me most suspicious is that since it's not on the official Microsoft organization, there was nothing to distinguish @msftenhanceprovenance from an account/bot anyone could have created.

The thing which convinced me of its legitimacy was [this link](https://github.com/microsoft/Secure-Supply-Chain/#enhancing-provenance-information-in-oss-software-supply-chains) to @msftenhanceprovenance, found on an official Microsoft repo (although the link has a typo which breaks it).

Short of hosting this project on the official Microsoft org, it would be nice if the PR could include a link so something _on the official Microsoft account_ which links back to this project so that people would instantly understand that the PRs are very unlikely to be malicious.

Thanks! <3

**Comment by iarna** [comment]
Yeah, you're right, that wording could improve a lot of improvement.: We're looking at adjusting the language there to be something like:

> Hi there!
> 
> This change adds the `repository` property to your `package.json` -- having this available provides a number of benefits to security tooling. For example, it allows for greater trust by checking for signed commits, contributors to a release and validating history with the project. It also allows for comparison between the source code and the published artifact in order to detect attacks on authors during the publication process.
> 
> We validate that we're making a PR against the correct repository by comparing the source code in your release tag to the published artifact on npmjs.org.
> 
> This change is provided by a team at Microsoft -- we're happy to answer any questions you may have. (Members of this team include @s-tuli and @iarna.) If you would prefer that we not make these sorts of PRs to projects you maintain, please just say. If you'd like to learn more about what we're doing here, we've prepared a document talking about both this project and some our other activities around supply chain security here: [microsoft/Secure-Supply-Chain](https://github.com/microsoft/Secure-Supply-Chain#readme)
> 
> This PR provides repository metadata for the following packages:
>		* `<npm package name>`

**Comment by rreusser** [comment]
Thanks for the prompt reply. The above sounds great! And the preface at the top of the secure-supply-chain readme helps out a lot. ❤️
